lib/ui/screens/contact_list_screen.dart

```diff
@@ -13,103 +14,110 @@ class ContactListScreen extends StatefulWidget {
+      //         )
+      // );
+  }
 
 
-    ListTile contactTile(String contactName, String phone, int index)
-    {
-        return 
-        ListTile(   
-        title: Text(contactName),
-        subtitle: Text(phone),
-        trailing: PopupMenuButton(
-            itemBuilder: (context) {
+  ListTile contactTile(String contactName, String phone, int index) {
+    return
+      ListTile(
+          title: Text(contactName),
+          subtitle: Text(phone),
+          trailing: PopupMenuButton(
+              itemBuilder: (context) {
                 return [
-                    PopupMenuItem(
-                        value: "edit",
-                        child: Text("Edit"),
-                    ), 
-                    PopupMenuItem(
-                        value: "delete",
-                        child: Text("Delete"),
-                    ),
-                ]; 
-            },
-            onSelected: (String value) => _manageContact(value, index),
-        ),
-        onTap: () => _navigateDetail(index)
-        ); 
-    }
+                  PopupMenuItem(
+                    value: "edit",
+                    child: Text("Edit"),
+                  ),
+                  PopupMenuItem(
+                    value: "delete",
+                    child: Text("Delete"),
+                  ),
+                ];
+              },
+              onSelected: (String value) {} // manageContact(value, index),
+          ),
+          onTap: () {} //navigateDetail(index)
+      );
+  }
 
-    _gotoAddNewContact(){
-        Navigator.push(
-            context,
-            MaterialPageRoute(builder: (context) => NewContactScreen())
-        );
-    }
-    
-    @override
-    Widget build(BuildContext context) {
-        return Scaffold(
-            appBar: AppBar(
-                title: Text("Contact List"),
-                actions: [
-                    IconButton(icon: Icon(Icons.add_circle_outline_sharp), onPressed: _gotoAddNewContact)],
-            ),
-            body: Container(
-                padding: EdgeInsets.all(20), 
-                child: ListView.builder(
-                    itemCount: contactList.length,
-                    itemBuilder: (context, index){
-                        String contactName = contactList[index].firstName + " " + contactList[index].lastName;
-                        String phone = contactList[index].phoneNumber; 
+  _gotoAddNewContact(){
+      Navigator.push(
+          context,
+          MaterialPageRoute(builder: (context) => NewContactScreen())
+      );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: Text("Contact List"),
+          actions: [
+            IconButton(icon: Icon(Icons.add_circle_outline_sharp),
+                onPressed: _gotoAddNewContact)
+          ],
+        ),
+        body: Container(
+            padding: EdgeInsets.all(20),
+            child: WatchBoxBuilder(
```

Comment:
'WatchBoxBuilder' is deprecated and shouldn't be used. Use [ValueListenableBuilder] and `box.listenable()` instead.
Try replacing the use of the deprecated member with the replacement.